### ISSUE TITLE
not wholly fatal ValueError in production

### ISSUE THREAD

**Issue by bensteinberg** [harvard-lil/dostuff#30]
In production, most but not all POSTs to `new_event` get a 500:

> ValueError: The future belongs to a different loop than the one specified as the loop argument

but the message is in fact successfully sent to the websocket. 

One approach to examining this would be to set up a postgres DB container in dev.

**Comment by bensteinberg** [comment]
```
2023-03-13 17:48:24,300 ERROR    Internal Server Error: /new_event
Traceback (most recent call last):
  File "/app/.heroku/python/lib/python3.9/site-packages/asgiref/sync.py", line 486, in thread_handler
    raise exc_info[1]
  File "/app/.heroku/python/lib/python3.9/site-packages/django/core/handlers/exception.py", line 43, in inner
    response = await get_response(request)
  File "/app/.heroku/python/lib/python3.9/site-packages/django/core/handlers/base.py", line 253, in _get_response_async
    response = await wrapped_callback(
  File "/app/.heroku/python/lib/python3.9/site-packages/asgiref/sync.py", line 448, in __call__
    ret = await asyncio.wait_for(future, timeout=None)
  File "/app/.heroku/python/lib/python3.9/asyncio/tasks.py", line 442, in wait_for
    return await fut
  File "/app/.heroku/python/lib/python3.9/site-packages/asgiref/current_thread_executor.py", line 22, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/app/.heroku/python/lib/python3.9/site-packages/asgiref/sync.py", line 490, in thread_handler
    return func(*args, **kwargs)
  File "/app/.heroku/python/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
  File "/app/.heroku/python/lib/python3.9/site-packages/django/views/generic/base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "/app/.heroku/python/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/app/.heroku/python/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/app/.heroku/python/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/app/.heroku/python/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/app/./main/views.py", line 58, in post
    async_to_sync(closing_send)(
  File "/app/.heroku/python/lib/python3.9/site-packages/asgiref/sync.py", line 240, in __call__
    return call_result.result()
  File "/app/.heroku/python/lib/python3.9/concurrent/futures/_base.py", line 439, in result
    return self.__get_result()
  File "/app/.heroku/python/lib/python3.9/concurrent/futures/_base.py", line 391, in __get_result
    raise self._exception
  File "/app/.heroku/python/lib/python3.9/site-packages/asgiref/sync.py", line 306, in main_wrap
    result = await self.awaitable(*args, **kwargs)
  File "/app/./main/views.py", line 52, in closing_send
    await channel_layer.close_pools()
  File "/app/.heroku/python/lib/python3.9/site-packages/channels_redis/core.py", line 498, in close_pools
    await self.wait_received()
  File "/app/.heroku/python/lib/python3.9/site-packages/channels_redis/core.py", line 508, in wait_received
    await asyncio.wait(self.receive_cleaners[:])
  File "/app/.heroku/python/lib/python3.9/asyncio/tasks.py", line 411, in wait
    fs = {ensure_future(f, loop=loop) for f in fs}
  File "/app/.heroku/python/lib/python3.9/asyncio/tasks.py", line 411, in <setcomp>
    fs = {ensure_future(f, loop=loop) for f in fs}
  File "/app/.heroku/python/lib/python3.9/asyncio/tasks.py", line 671, in ensure_future
    raise ValueError('The future belongs to a different loop than '
ValueError: The future belongs to a different loop than the one specified as the loop argument
```

**Comment by bensteinberg** [comment]
Closed in [f5df460](https://github.com/harvard-lil/dostuff/commit/f5df460b0d5ea143ec034cfcefe298e656a22516)

There's no need to `await` `channel_layer.close_pools()`; the client now gets 200 every time, even though we see some 

```
2023-03-15 17:47:30,672 ERROR    Task exception was never retrieved
future: <Task finished name='Task-2433' coro=<Connection.disconnect() done, defined at /root/.cache/pypoetry/virtualenvs/dostuff-9TtSrW0h-py3.9/lib/python3.9/site-packages/redis/asyncio/connection.py:723> exception=RuntimeError("Task <Task pending name='Task-2433' coro=<Connection.disconnect() running at /root/.cache/pypoetry/virtualenvs/dostuff-9TtSrW0h-py3.9/lib/python3.9/site-packages/redis/asyncio/connection.py:736>> got Future <Future pending> attached to a different loop")>
Traceback (most recent call last):
  File "/root/.cache/pypoetry/virtualenvs/dostuff-9TtSrW0h-py3.9/lib/python3.9/site-packages/redis/asyncio/connection.py", line 736, in disconnect
    await self._writer.wait_closed()  # type: ignore[union-attr]
  File "/usr/local/lib/python3.9/asyncio/streams.py", line 359, in wait_closed
    await self._protocol._get_close_waiter(self)
RuntimeError: Task <Task pending name='Task-2433' coro=<Connection.disconnect() running at /root/.cache/pypoetry/virtualenvs/dostuff-9TtSrW0h-py3.9/lib/python3.9/site-packages/redis/asyncio/connection.py:736>> got Future <Future pending> attached to a different loop
```

on the backend in development and production.
